migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,165 @@
+
+For more information, see xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-about_configuring-networkpolicy-plugin[About network policy].
+
+[discrete]
+==== Encrypting traffic between hosts
+
+In {product-title} 3.11, you could use IPsec to encrypt traffic between hosts. {product-title} 4.2 does not support IPsec. It is recommended to use Red Hat OpenShift Service Mesh to enable mutual TLS between services.
+
+For more information, see xref:../service_mesh/service_mesh_arch/understanding-ossm.adoc#understanding-ossm[Understanding Red Hat OpenShift Service Mesh].
+
+[id="migration-preparing-logging"]
+=== Logging considerations
+
+Review the logging changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Deploying cluster logging
+
+{product-title} 4 provides a simple deployment mechanism for cluster logging, by using a Cluster Logging custom resource. Once deployed, the cluster logging experience is the same as it was in {product-title} 3.11.
+
+For more information, see xref:../logging/cluster-logging-deploying-about.adoc#cluster-logging-deploying-about_cluster-logging-deploying-about[About deploying and configuring cluster logging].
+
+[discrete]
+==== Aggregated logging data
+
+You cannot transition your aggregate logging data from {product-title} 3.11 into your new {product-title} 4 cluster. You must start fresh with the new cluster.
+
+For more information, see xref:../logging/cluster-logging.adoc#cluster-logging-about_cluster-logging[About cluster logging].
+
+[id="migration-preparing-security"]
+=== Security considerations
+
+Review the security changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Unauthenticated access to discovery endpoints
+
+In {product-title} 3.11, an unauthenticated user could access the discovery endpoints (for example, [x-]`/api/*` and [x-]`/apis/*`). In {product-title} 4.2, unauthenticated access to the discovery endpoints is no longer allowed. If you do need to allow unauthenticated access, you can configure the RBAC settings as necessary.
+
+// TODO: Anything to xref to, or additional details?
+
+[discrete]
+==== Identity providers
+
+The request header identity provider in {product-title} 4.2 requires mutual TLS, where in {product-title} 3.11 it did not. The configuration of the OpenID Connect identity provider was simplified in {product-title} 4.2. It now obtains data, which previously had to specified in {product-title} 3.11, from the provider's `/.well-known/openid-configuration` endpoint.
```

Comment:
@stlaz Can you review to make sure my rewording is still accurate? Thanks!

Preview: https://migration-guide--ocpdocs.netlify.com/openshift-enterprise/latest/migration/planning-migration-3-to-4.html#migration-preparing-security

Reply:
LGTM now